kit/azure/organization-hierarchy/modules/policies/variables.tf

```diff
@@ -0,0 +1,104 @@
+variable "management_group_id" {
```

Comment:
it feels like this is one module that can deploy either (combination) of definitions, assignments, sets

there's probably some reason to keep the reuse high here instead of splitting up into three different modules?


also could use a README.md (or a mention in the `kit/azure/organization-hierarchy/README.md` that this module is just a utility implementing a simple machinery for defining azure policy definitions, sets, assignments as json files on disk? Which makes it compatible with copy/pasting policies from CAF or the Azure console (for modifying/starting from built-in policies)